Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -68,6 +57,19 @@ the token returned by ``kubeadm init``:
     each node has an ``InternalIP`` which is assigned to a device with the same
     name on each node.
 
+For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
+by using the following commands below. **Careful:** Be aware that this will break
+existing connections. It will also stop service related traffic until the Cilium
```

Comment:
s/existing connections/existing service connections/

Reply:
From kp->kpr agree, the other way round it would potentially be more general (since we also SNAT regular connections to avoid port conflicts). But for this one, makes sense to clarify, ack.